core/trino-main/src/test/java/io/trino/execution/TestNodeSchedulerConfig.java

```diff
@@ -60,7 +59,6 @@ public void testExplicitPropertyMappings()
                 .put("node-scheduler.splits-balancing-policy", "node")
                 .put("node-scheduler.optimized-local-scheduling", "false")
                 .put("node-scheduler.allowed-no-matching-node-period", "1m")
-                .put("node-scheduler.allocator-type", "fixed_count")
```

Comment:
Does TestNodeSchedulerConfig pass after first commit?

Reply:
It does :)